### PR TITLE
created docker file for running commands

### PR DESCRIPTION
**Pull request by charlycastro**
I was not able  to push the docker image

anyone able to help me with that?
```
docker push wizelineacademy/wz_terraform:latest
The push refers to repository [docker.io/wizelineacademy/wz_terraform]
21d08a19dc3c: Preparing 
b4ed7b359ab0: Preparing 
302cd06a7b42: Preparing 
3601687ed993: Preparing 
62960d942fc2: Preparing 
8a13ac9ece63: Waiting 
103a15f3ef0b: Waiting 
aabe8fddede5: Waiting 
bcf2f368fe23: Waiting 
denied: requested access to the resource is denied```

### CODE REVIEW

File: python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ENV TERRAFORM_VERSION 0.11.13
+
+RUN apk --update --no-cache add curl && cd /usr/local/bin && \
+    curl https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -o terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt && mkdir -p /root/.aws && touch /root/.aws/credentials
+ENTRYPOINT ["/bin/ash"]
```
Comment (dancortes94) [inline python/Dockerfile:12]: Think this is a typo, shouldn't it be `["/bin/bash"]`?

**Review by dancortes94 — CHANGES_REQUESTED** [review]

File: python/Dockerfile (hunk 1 above)
Comment (luisc009) [reply python/Dockerfile:12]: ash is another shell that comes installed by default in the alpine images.

File: python/README.md [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-`source ./source.sh`
-
-To initialize a Terraform backend on a specific lesson run:
+# Configuration Steps
+These steps helps you configure your AWS credentials into the requiered files and initialize terraform backends.
+
+1. cd root of your git repo
+2. run the following command `docker run -it -v ${PWD}:/terraform wizelineacademy/wz_terraform:latest`
+3. Run `cd python`
+4.create a `variables.ini` file, following the same format as the `variables.ini.example`. Here, you'll add your AWS credentials, profile name and environment.
```
Comment (luisc009) [inline python/README.md]: Could you change `4.c` to `4. C`

**Comment by luisc009** [comment]
~Unable to download the image~
image

~Seems the image is set as private.~

Just saw the comment, sowy :D

File: python/Dockerfile (hunk 1 above)
Comment (charlycastro) [reply python/Dockerfile:12]: this ^ hehe

File: python/README.md (hunk 2 above)
Comment (charlycastro) [reply python/README.md]: update

File: python/Dockerfile (hunk 1 above)
Comment (dancortes94) [reply python/Dockerfile:12]: Didn't know that, hadn't used alpine really. Thanks 😄

File: python/README.md [hunk 3]
```diff
@@ -1,25 +1,13 @@
...
-
-After running this command, you need to run the following command:
-`source ./source.sh`
-
-To initialize a Terraform backend on a specific lesson run:
+# Configuration Steps
+These steps helps you configure your AWS credentials into the requiered files and initialize terraform backends.
+
+1. cd root of your git repo
+2. run the following command `docker run -it -v ${PWD}:/terraform wizelineacademy/wz_terraform:latest`
```
Comment (dancortes94) [inline python/README.md]: Can you please change this to `2. Run`?

File: python/README.md [hunk 4]
```diff
@@ -1,25 +1,13 @@
...
-`python3 awsconfig.py`
-
-After running this command, you need to run the following command:
-`source ./source.sh`
-
-To initialize a Terraform backend on a specific lesson run:
+# Configuration Steps
+These steps helps you configure your AWS credentials into the requiered files and initialize terraform backends.
+
+1. cd root of your git repo
```
Comment (dancortes94) [inline python/README.md]: Should we change this to `1. Change directory` so we can comply with the style on the rest of the README?

**Review by dancortes94 — APPROVED** [review]
LGTM

**Pull request merged**